src/sweetalert2.js

```diff
@@ -1176,6 +1176,52 @@ sweetAlert.init = function() {
     sweetAlert.resetValidationError();
   };
 
+  /*
+   *  @summary IOS input focus fix
+   *  @var {iosfix}, {head} 
+   *  @description creating <style> tag with ios on focus fix
+   *  @author yeomann
+  */
+  var iosfix;
+  iosfix = document.createElement('style');
+  iosfix.type = "text/css";
+  iosfix.id = "iosFix"
+  iosfix.innerHTML = "html,body {height: 100%; position: relative; }";
```

Comment:
do you mind addition of `overflow: hidden;` under html,body{}, i am pretty sure it will work.
PS: it worked for me on my iphone. how come it didn't work on your ipad! interesting.